### PR TITLE
Misc documentation improvements

### PR DESCRIPTION
**Pull request by mssalvatore**
# What does this PR do?

Miscellaneous documentation fixes.

## PR Checklist
* [ ] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [ ] Was the CHANGELOG.md updated to reflect the changes?
* [ ] Was the documentation framework updated to reflect the changes?
* [ ] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [ ] Added relevant unit tests?
* [ ] Do all unit tests pass?
* [ ] Do all end-to-end tests pass?
* [ ] Any other testing performed?
    > Tested by {Running the Monkey locally with relevant config/running Island/...}
* [ ] If applicable, add screenshots or log transcripts of the feature working

### CODE REVIEW

File: docs/content/reference/exploiters/PowerShell.md [hunk 1]
```diff
@@ -16,45 +16,46 @@ and [Windows Remote
...
    blank password.
 
    In order for the attacker to connect with a blank password, the victim must
    have enabled basic authentication, http and no encryption.
 
 1. **Brute force usernames with cached password (Windows attacker only)** - The
    exploiter will attempt to log into the victim machine using each username
-   set in the [configuration]({{< ref "/usage/configuration/credentials"
-   >}}) and the current user's cached password.
+   set in the [configuration](/usage/configuration/credentials") and the
```
Comment (cakekoa) [inline docs/content/reference/exploiters/PowerShell.md:44]:
```suggestion
   set in the [configuration](/usage/configuration/credentials) and the
```

File: docs/content/reference/exploiters/SMBExec.md [hunk 2]
```diff
@@ -6,4 +6,4 @@ tags: ["exploit", "windows"]
 ---
 ### Description
 
-This exploit brute forces machines using credentials provided by the user (see [configuration]({{< ref "/usage/configuration" >}}) for instructions) and hashes gathered from infected systems by Mimikatz.
+This exploit brute forces machines using credentials provided by the user (see [configuration](/usage/configuration) for instructions) and hashes gathered from infected systems by Mimikatz.
```
Comment (cakekoa) [inline docs/content/reference/exploiters/SMBExec.md:9]: This links to the high-level configuration page, wheras the elsewhere we link directly to the credentials page. Should this link to the credentials page instead?

File: docs/content/reference/exploiters/SNMP.md [hunk 3]
```diff
@@ -14,4 +14,4 @@ If a community is configured with read-write access, the machine is vulnerable t
 
 In versions 1 and 2c of the SNMP protocol the only means of authentication is the community string. In these versions of SNMP, community strings are sent as plaintext in all requests. Version 3 adds user-based authentication and encryption to the SNMP protocol.
 
-This exploiter uses version 2c of the SNMP protocol. It brute-forces machines by using usernames and passwords provided by the user (see [configuration]({{< ref "/usage/configuration" >}}) for instructions) as SNMP community strings.
+This exploiter uses version 2c of the SNMP protocol. It brute-forces machines by using usernames and passwords provided by the user (see [configuration](/usage/configuration) for instructions) as SNMP community strings.
```
Comment (cakekoa) [inline docs/content/reference/exploiters/SNMP.md:17]: [Same comment as on SMBExec](https://github.com/guardicore/monkey/pull/4198/files#r1638318764). Looks like SSHexec and WMIExec link to this page as well

Comment (mssalvatore) [reply docs/content/reference/exploiters/SNMP.md:17]: Same as above.

**Review by cakekoa — APPROVED** [review]

**Pull request merged**